content/hacking-atom/sections/publishing.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Publishing"
+---
+
+### Publishing
+
+Atom bundles a command line utility called `apm` which we first used back in [Command Line](/using-atom/sections/atom-packages/#command-line) to search for and install packages via the command line. The `apm` command can also be used to publish Atom packages to the public registry and update them.
+
+#### Prepare Your Package
+
+There are a few things you should double check before publishing:
+
+* Your `package.json` file has `name`, `description`, and `repository` fields.
+* Your `package.json` file has a `version` field with a value of  `"0.0.0"`.
+* Your `package.json` file has an `engines` field that contains an entry for Atom such as: `"engines": {"atom": ">=1.0.0 <2.0.0"}`.
+* Your package has a `README.md` file at the root.
+* Change the `repository` url in the `package.json` file to match the URL of your repository.
+* Your package is in a Git repository that has been pushed to [GitHub](https://github.com). Follow [this guide](http://guides.github.com/overviews/desktop) if your package isn't already on GitHub.
+
+#### Publish Your Package
+
+Before you publish a package it is a good idea to check ahead of time if a package with the same name has already been published to [the atom.io package registry](https://atom.io/packages). You can do that by visiting `https://atom.io/packages/your-package-name` to see if the package already exists. If it does, update your package's name to something that is available before proceeding.
+
+Now let's review what the `apm publish` command does:
+
+1. Registers the package name on atom.io if it is being published for the first time.
+2. Updates the `version` field in the `package.json` file and commits it.
+3. Creates a new [Git tag](http://git-scm.com/book/en/Git-Basics-Tagging) for the version being published.
+4. Pushes the tag and current branch up to GitHub.
+5. Updates atom.io with the new version being published.
+
+Now run the following commands to publish your package:
+
+``` command-line
+$ cd ~/github/my-package
```

Comment:
Should we just change it to `$ cd path-to-your-package`. The `/github/` directory gets used by the Package Generator from the Word Count example, but that might not be the case for everyone.